setup.py

```diff
@@ -32,9 +32,9 @@
         'nbgrader.exporters'
     ],
     package_data={
-        '': [
-            'nbextensions/*.js',
-            'nbextensions/*.css'
+        'nbgrader': [
```

Comment:
Is it necessary to move these into an additional subfolder? Is it just to keep the `nbextensions` folder in the profile less cluttered? If so, then is it possible to just create the folder at install time, rather than having an extra directory in the nbgrader repository itself?